rust/src/concept/value.rs

```diff
@@ -539,9 +569,53 @@ impl FromStr for Duration {
     }
 }
 
+/// ISO-8601 compliant representation of a duration
 impl fmt::Display for Duration {
```

Comment:
The more I copy from the server for Durations, the more I want `typedb_common` =( (sarcasm (or not))